core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -297,22 +298,25 @@ final class ShuffleBlockFetcherIterator(
 
     val fallback = FallbackStorage.FALLBACK_BLOCK_MANAGER_ID.executorId
     for ((address, blockInfos) <- blocksByAddress) {
-      if (Seq(blockManager.blockManagerId.executorId, fallback).contains(address.executorId)) {
+      // TODO (wuyi): For HDFS-like filesytem, shuffle fetch should be always considered as
```

Comment:
Please use IDed TODO like `TODO(SPARK-XXX)` instead of person id because it's a community code base.

Reply:
And, some word change and typo fix.
- `HDFS-like` -> `HDFS-compatible`
- `filesytem` -> `filesystem`

Reply:
Thanks for the guidance!